src/main/java/io/appium/java_client/windows/WindowsDriver.java

```diff
@@ -79,6 +80,28 @@ public WindowsDriver(HttpClient.Factory httpClientFactory, Capabilities capabili
                 capabilities, PLATFORM_NAME, AUTOMATION_NAME));
     }
 
+    /**
+     * Creates a new instance based on the given ClientConfig and {@code capabilities}.
+     * The HTTP client is default client generated by {@link HttpCommandExecutor#getDefaultClientFactory}.
+     *
+     * Example:
+     *
+     *  ClientConfig clientConfig = ClientConfig.defaultConfig()
+     *    .baseUri(URI.create("WebDriver URL"))
+     *    .proxy(new Proxy(Proxy.Type.HTTP, new InetSocketAddress("your.proxy.ip", 8080)))
+     *    .readTimeout(Duration.ofMinutes(5));
+     *    options = WindowsOptions options = new WindowsOptions();
```

Comment:
extra leading spaces here
Consider formatting the code section though: https://reflectoring.io/howto-format-code-snippets-in-javadoc/

Reply:
changed to use pre